src/tools/web/clusterd/README.md

```diff
@@ -0,0 +1,32 @@
+# clusterd
```

Comment:
Please add a proper README.md explaining what it does, how it does it and so on..